fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -4683,27 +4683,49 @@ public void addOrReplaceAutoIncrementIdByTableId(Long tableId, Long id) {
         }
     }
 
+    // For code reuse
+    public static int getDatabaseSavedJsonNum(Database database) {
+        return 2 + database.getTableNumber() * getTableSavedJsonNum();
+    }
+
+    // For code reuse
+    public static void saveDatabase(SRMetaBlockWriter writer, Database database)
+            throws IOException, SRMetaBlockException {
+        writer.writeJson(database);
+        writer.writeJson(database.getTables().size());
+        List<Table> tables = database.getTables();
+        for (Table table : tables) {
+            saveTable(writer, table);
+        }
+    }
+
+    // For code reuse
```

Comment:
comments like this won't provide any help

Reply:
just to remind others not to delete the function